Makefile

```
@@ -121,12 +121,12 @@ HTML_DST = \
 
 ## * install-rmd-deps : Install R packages dependencies to build the RMarkdown lesson
 install-rmd-deps:
-	Rscript -e 'source("bin/dependencies.R"); install_dependencies(identify_dependencies())'
+	@bin/install_r_deps.sh
```

```suggestion
	@${SHELL} bin/install_r_deps.sh
```

I suggest avoid relying on executable bit + shebang in the script -- we might run into issues on Windows platforms. Instead, we can use shell to execute the script.